packages/containers/src/ActionBar/ActionBar.container.js

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes } from 'react';
+
+import { ActionBar as Component } from 'react-talend-components';
+import { getActionsProps } from '../actionAPI';
+
+function getActions(context, idOrInfo, model) {
+	if (typeof idOrInfo === 'string') {
+		return getActionsProps(context, idOrInfo, model);
+	} else if (idOrInfo.displayMode === 'splitDropdown') {
+		return Object.assign({
+			displayMode: idOrInfo.displayMode,
+			items: getActionsProps(context, idOrInfo.items, model),
+		}, getActionsProps(context, idOrInfo.name, model));
+	} else if (idOrInfo.displayMode === 'btnGroup') {
+		return {
+			displayMode: idOrInfo.displayMode,
+			actions: getActionsProps(context, idOrInfo.actions, model),
+		};
+	}
+	return undefined;
+}
+
+
+class ActionBar extends React.Component {
```

Comment:
Linter error : `Component should be written as a pure function  react/prefer-stateless-function`